pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -205,15 +205,19 @@ OFFSET_TO_PERIOD_FREQSTR: dict = {
     "ns": "ns",
     "H": "H",
     "Q": "Q",
-    "A": "A",
+    "Y": "Y",
     "W": "W",
     "ME": "M",
-    "Y": "A",
-    "BY": "A",
-    "YS": "A",
-    "BYS": "A",
+    "BY": "Y",
+    "YS": "Y",
+    "BYS": "Y",
+}
+OFFSET_DEPR_FREQSTR: dict[str, str]= {
+    "M": "ME",
 }
```

Comment:
I'm confused, why is `'M' -> `'ME'` in `OFFSET_DEPR_FREQSTR`, but `'A' -> 'Y'` is in `DEPR_ABBREVS`?

Reply:
`'M' -> 'ME'` in `OFFSET_DEPR_FREQSTR` because we deprecate `'M'` only for offsets, but we still have it for period.
` 'A' -> 'Y'` is in `DEPR_ABBREVS` because we deprecate` 'A' ` from everywhere in favour of `'Y'.` After this, we will rename `'Y'` to `'YE'` for offsets, and ` 'Y '-> 'YE'` will be in `OFFSET_DEPR_FREQSTR`.

Reply:
ah ok, thanks for explaining - all good then 👌